### PR TITLE
add identifier unicode deprecation example

### PR DESCRIPTION
**Pull request by nadja-muller**

### CODE REVIEW

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -923,6 +923,20 @@ The product's default behavior of using a cost-based IDP search algorithm when c
...
+Query::
++
+[source,cypher]
+----
+RETURN 1 as my\u0085identifier
+----
+Description of the returned code::
+The unicode `\\u0061` is deprecated in unescaped identifiers.
+Suggestions for improvement::
+If you want to continue to use this unicode, please escape the identifier instead.
```
Comment (Lojjs) [inline modules/ROOT/pages/notifications/all-notifications.adoc:937]:
```suggestion
Description of the returned code::
The unicode `\\u0061` is deprecated in unescaped identifiers.
If you want to continue to use this unicode, please escape the identifier instead.
```

I know this is not super clear in the instruction, 'Description of the returned code' is supposed to be the same description as in Neo4j, and 'Suggestions for improvement' is any other suggestion which you might want to provide but doesn't fit in the description (e.g. too long text or need an example).

**Review by Lojjs — CHANGES_REQUESTED** [review]
I like the message, just a formatting suggestion. I think more people got confused by this, so we should discuss how we can make this clearer when we change the process @renetapopova

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 2]
```diff
@@ -923,6 +923,20 @@ The product's default behavior of using a cost-based IDP search algorithm when c
...
+.Using the unicode \u0085 in an unescaped identifier
+====
+Query::
++
+[source,cypher]
+----
+RETURN 1 as my\u0085identifier
+----
+Description of the returned code::
+The unicode `\\u0061` is deprecated in unescaped identifiers.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:935]: This sounds like only that particular Unicode `\\u0061` is deprecated. I guess all Unicodes must be escaped when used as identifiers.

Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:935]: I somehow can't understand what is deprecated, as the Unicode in the example is \u0085; in the query, it is `my\u0085identifier`, and in the description - `\\u0061`. 🙈

Comment (nadja-muller) [reply modules/ROOT/pages/notifications/all-notifications.adoc]: No, you can still use other unicodes. `RETURN 1 as my\u0061identifier` is parsed as `RETURN 1 as myaidentifier` at is valid.

Comment (nadja-muller) [reply modules/ROOT/pages/notifications/all-notifications.adoc]: Right....copy paste mistake, it should be \u0085, with a single backslash.

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 3]
```diff
@@ -923,6 +923,19 @@ The product's default behavior of using a cost-based IDP search algorithm when c
...
+====
+Query::
++
+[source,cypher]
+----
+RETURN 1 as my\u0085identifier
+----
+Description of the returned code::
+The unicode `\u0085` is deprecated in unescaped identifiers.
+If you want to continue to use this unicode, please escape the identifier instead.
```
Comment (renetapopova) [inline modules/ROOT/pages/notifications/all-notifications.adoc:936]:
```suggestion
```

**Review by renetapopova — APPROVED** [review]
From an editorial perspective, it looks good to me. Do we need the `\` before the Unicode and the identifier?

**Comment by nadja-muller** [comment]
In the preview it looked like it's not printing the backticks otherwise and formats it differently. You can try it out in the comment section here as well:`test` vs \`test`

**Review by Lojjs — APPROVED** [review]

**Pull request merged**